ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -342,6 +356,240 @@ objects:
+                'Comment': 'add %s -> %s' % (record, ip),
+                'Changes': [
+                {
+                    'Action': 'UPSERT',
+                    'ResourceRecordSet': {
+                        'Name': record,
+                        'Type': 'A',
+                        'TTL': 60,
+                        'ResourceRecords': [{'Value': ip}]
+                    }
+                }]
+        })
+        PYTHON_EOF
+          for i in "${!NEW_MASTER_IPS[@]}"; do
+            ETCD_NAME="etcd-${i}.${CLUSTER_NAME}.${BASE_DOMAIN}"
+            python /tmp/update_route_53.py "${ETCD_NAME}" "${NEW_MASTER_IPS[$i]}"
+          done
+
+          echo "Run etcd-signer"
+          SURVIVING_MASTER_NODE_SHORT=${SURVIVING_MASTER_NODE%%.*}
+          curl -O https://raw.githubusercontent.com/hexfusion/openshift-recovery/master/manifests/kube-etcd-cert-signer.yaml.template
+          sed "s;__MASTER_HOSTNAME__;${SURVIVING_MASTER_NODE_SHORT};g" kube-etcd-cert-signer.yaml.template > kube-etcd-cert-signer.yaml
+          retry 10 oc create -f kube-etcd-cert-signer.yaml
+          retry 10 oc get pod/etcd-signer -n openshift-config -o name
+          retry 10 oc wait pod/etcd-signer -n openshift-config --for condition=ready
+
+          echo "Grow etcd cluster to full membership"
+          SURVIVING_MASTER_IP=$(oc get nodes ${SURVIVING_MASTER_NODE} -o jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
+          SETUP_ETCD_ENVIRONMENT=$(oc adm release info --image-for setup-etcd-environment)
+          KUBE_CLIENT_AGENT=$(oc adm release info --image-for kube-client-agent)
+          MASTERS=($(oc -n openshift-machine-api \
+            get machines \
+            -l machine.openshift.io/cluster-api-machine-role=master \
+            -o jsonpath='{range .items[*]}{.status.addresses[?(@.type=="InternalDNS")].address}{"\n"}{end}'))
+          for master in ${MASTERS[@]}
+          do
+            if [[ "${master}" == ${SURVIVING_MASTER_NODE} ]]; then continue; fi
+            retry 10 \
+              scp -o StrictHostKeyChecking=no -o ProxyCommand="ssh -A -o StrictHostKeyChecking=no \
+                  -o ServerAliveInterval=30 -W %h:%p core@${BASTION_HOST}" \
+                  /tmp/install_recovery_scripts.sh core@${master}:/tmp
+            bastion_ssh core@${master} "sudo -i /tmp/install_recovery_scripts.sh"
+            bastion_ssh core@${master} "sudo -i env SETUP_ETCD_ENVIRONMENT=${SETUP_ETCD_ENVIRONMENT} KUBE_CLIENT_AGENT=${KUBE_CLIENT_AGENT} /usr/local/bin/etcd-member-recover.sh ${SURVIVING_MASTER_IP}"
+          done
+
+          for master in ${MASTERS[@]}
+          do
+            retry 10 oc get pod/etcd-member-${master} -n openshift-etcd -o name
+            oc wait pod/etcd-member-${master} -n openshift-etcd --for condition=Ready
+          done
+
+          #echo "Verify etcdctl member list returns 3 members"
+
+          #echo "Verify kubernetes svc points to 3 pods"
+
+          # Verify no alerts are firing
```

Comment:
Probably drop these commented-out placeholders and file issues if you need a way to track future plans.

Reply:
Fixed in 143155551